govcd/org_test.go

```diff
@@ -174,6 +175,86 @@ func (vcd *TestVCD) Test_Admin_GetVdcByName(check *C) {
 	check.Assert(err, IsNil)
 }
 
+// Tests org function GetVDCByName with the vdc specified
+// in the config file. Then tests with a vdc that doesn't exist.
+// Fails if the config file name doesn't match with the found vdc, or
+// if the invalid vdc is found by the function.  Also tests an vdc
+// that doesn't exist. Asserts an error if the function finds it or
+// if the error is not nil.
+func (vcd *TestVCD) Test_CreateVdc(check *C) {
+	if vcd.skipAdminTests {
+		check.Skip("Configuration org != 'Sysyem'")
+	}
+
+	results, err := vcd.client.QueryWithNotEncodedParams(nil, map[string]string{
```

Comment:
Before creating a VDC, we should check whether the VDC exists already. If yes, we should either delete it and create a new one, or exit gracefully with a meaningful error message.